### DISCUSSION TITLE
Stylesheet component for React

### DISCUSSION THREAD

**Discussion by frolic** [tw-in-js/twind#154] [Ideas]
It'd be nice if ya'll exposed some sort of `<Stylesheet>` component that contains the injected styles so that we could choose where/when to mount this in the DOM.

This might help address some of the things we chatted about in https://github.com/tw-in-js/twind/discussions/141. It'd also help in situations where we want to add/inject global styles but only on pages using Twind, not in others.

**Comment by frolic** [comment]
I think this is roughly what I was aiming for: https://codesandbox.io/s/strange-bose-3e51c?file=/src/Example.js

**Reply by sastan** [reply]
Sorry. Totally missed this one. Looking into it now.
